code-challenges.js

```diff
@@ -29,20 +48,79 @@ var myString2 = "delta"
 
 
 
+myString1.split("") 
+// Step 1: I split the string so that each letter was a separate index
+var myString3 = myString1.split("")
+myString3.reverse()
+//When I tried to reverse myString1.split("") I kept getting the TypeError that myString1.reverse is not a function so I made a new variable to reverse the string, I think this is because the split turned the letters into an array. Is there a better method?
+myString3.join("")
+// After I reversed the array I joined it again as a string
+
+myString2.split("") 
+// I split the myString2 so that each letter was a separate index
+var myString4 = myString2.split("")
+myString4.reverse()
+//When I tried to reverse myString2.split("") I kept getting the TypeError that myString2.reverse is not a function so I made a new variable to reverse the string
+myString4.join("")
+// After I reversed the array I joined it again as a string
+
+var newArray = [myString3.join(""), myString4.join("")]
+console.log(newArray)
+//I was unsure if the intention of this problem was to leave the variables as separate strings or make an array with the output
```

Comment:
Glad you did both if you were unsure. Also feel free to ask that kind of question in the 2021d slack. The expected out put is separate strings and you can always fall back on and rely on the expected inputs and outputs. Solid work!